### DISCUSSION TITLE
状态栏网速错位

### DISCUSSION THREAD

**Discussion by zhanghaoyu0** [MonwF/customiuizer#387] [Q&A]
高级设置版本 23.10.30hyperos测试版1107 系统版本开发版11.4 ![Screenshot_2023-11-08-09-25-26-086_com.github.android.jpg](https://github.com/MonwF/customiuizer/assets/47391126/9a87afbd-73e5-4f68-b89d-486e5d89623b)

**Comment by MonwF** [comment]
网速怎么配置的？

**Reply by zhanghaoyu0** [reply]
![Screenshot_2023-11-08-09-30-16-238_name.monwf.customiuizer.jpg](https://github.com/MonwF/customiuizer/assets/47391126/311759a4-a965-4fa8-a371-8b896d9dac24)

**Comment by zhanghaoyu0** [comment]
还有diy时钟也不生效

**Reply by zhanghaoyu0** [reply]
![Screenshot_2023-11-08-09-32-02-310_name.monwf.customiuizer.jpg](https://github.com/MonwF/customiuizer/assets/47391126/c8c7ea4b-57fa-4e32-81fe-d14409619fc9)

时钟设置图

**Reply by MonwF** [reply]
时钟这个有没日志？

**Reply by zhanghaoyu0** [reply]
> 时钟这个有没日志？

不知道怎么生成日志  或者是在哪里看diy时钟不生效产生的日志

**Reply by MonwF** [reply]
在lsp管理器里，最下方右边第2个tab

**Reply by zhanghaoyu0** [reply]
> 在lsp管理器里，最下方右边第2个tab

![Screenshot_2023-11-08-09-44-17-059_com.github.android.jpg](https://github.com/MonwF/customiuizer/assets/47391126/775d52cd-db84-4054-90a8-c424c96e8bc4)

上传不上来   传蓝奏云盘可以吗

**Reply by MonwF** [reply]
可以的

**Reply by zhanghaoyu0** [reply]
> 可以的

https://www.123pan.com/s/9wuKVv-ccgk3.html  提取码：FjGV    



123盘

**Reply by MonwF** [reply]
看着没啥问题，你怎么配置的？

**Reply by zhanghaoyu0** [reply]
> 看着没啥问题，你怎么配置的？

不好意思 是我的问题  我把手机设置为24小时后  DIY时间就生效了

**Reply by MonwF** [reply]
意思是12小时或者自动就不生效？之前是怎么配置的？

**Comment by MonwF** [comment]
网速间隔、大小加粗这些都没适配，只是适配了简单的双排网速

**Reply by zhanghaoyu0** [reply]
忘记左右侧间距默认值多少了   我该回去试试

**Reply by zhanghaoyu0** [reply]
尝试调了下  不错位了  感谢
